src/System.Security.Cryptography.X509Certificates/tests/PfxTests.cs

```diff
@@ -133,6 +133,7 @@ public static void ReadECDsaPrivateKey_WindowsPfx()
         }
 
         [Theory, MemberData(nameof(BrainpoolCurvesPfx))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp1_0, "dotnet/corefx#12235")]
```

Comment:
It is passing today because we x-compile the X509Certificates project against ns1.6 but don't do that anymore with this set of changes. The tests require code\fixes added in the 4.1.1 version of that assembly (corefx 1.1.0 branch). Once ns1.6 gets updates the tests should work again.